uniswap_swap_detection/README.md

```diff
@@ -0,0 +1,31 @@
+# uniswapV3 Swap Detection Bot
+
+## Description
+
+ This bot detect swap that occurs on uniswap v3 pool 
+
+## Supported chain
+
+- Polygon 
+
+## Alerts
+Uniswap-1
+- It is fired when Uniswap swap is detected
+- it emmits the token address and the pool address 
+- Finding type is set to Info 
+- FInding severity is set to Low
+- Metadata includes
+   - sender: the address of the sender
+   - recipient: the address of the recipient
+   - token0: the address of the token deposited
+   - token1: the address of the token to be reeceived
+   - amount0: amount of token deposited
+   - amount1: amount of token to be received
```

Comment:
```suggestion
   - `sender`: the address of the sender
   - `recipient`: the address of the recipient
   - `token0`: the address of the token deposited
   - `token1`: the address of the token to be reeceived
   - `amount0`: amount of token deposited
   - `amount1`: amount of token to be received
```
Great to see the breakdown. Just a quick note that the properties of the metadata should be in ticks(`).